app/controllers/sail/settings_controller.rb

```diff
@@ -14,6 +14,12 @@ def update
       end
     end
 
+    def show
+      @setting = Sail::Setting.get(params[:name])
```

Comment:
We can use a local variable instead of an instance variable here, since no views need to access it.